getting-started/k8s/README.md

```diff
@@ -0,0 +1,93 @@
+# Getting Started with linkerd and Kubernetes
```

Comment:
I like the guide :)

It's a little confusing to have both this file and also https://github.com/BuoyantIO/linkerd-examples/blob/master/k8s/README.md
Even though this is the more generic guide, the other gob guide is more visible because it's under k8s/

One way to solve this is to move all the gob-specific stuff to it's own folder. So the top directories would be
`gob`
`getting-started`
`reqz`